From 80f2f8347c798577775bcbf162294d2bdf30ddaf Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Tue, 2 Jul 2013 13:38:49 +0900
Subject: [PATCH] Add check for below array bounds

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 c_src/double-conversion/fast-dtoa.cc |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/c_src/double-conversion/fast-dtoa.cc b/c_src/double-conversion/fast-dtoa.cc
index 1a0f823..b2f7b8d 100644
--- a/c_src/double-conversion/fast-dtoa.cc
+++ b/c_src/double-conversion/fast-dtoa.cc
@@ -253,6 +253,8 @@ static void BiggestPowerTen(uint32_t number,
   // number < (2^number_bits - 2) yet.
   while (number < kSmallPowersOfTen[exponent_plus_one_guess]) {
     exponent_plus_one_guess--;
+    if (exponent_plus_one_guess <= 0)
+      break;
   }
   *power = kSmallPowersOfTen[exponent_plus_one_guess];
   *exponent_plus_one = exponent_plus_one_guess;
-- 
1.7.10.4

