From c842cc1618e46e16307478a74833d18b3430826e Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Tue, 2 Jul 2013 13:05:54 +0900
Subject: [PATCH 1/2] Fix typo in utils.h

Signed-off-by: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
---
 c_src/double-conversion/utils.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/c_src/double-conversion/utils.h b/c_src/double-conversion/utils.h
index a19f85a..7fb7f04 100644
--- a/c_src/double-conversion/utils.h
+++ b/c_src/double-conversion/utils.h
@@ -300,7 +300,7 @@ class StringBuilder {
 //
 
 #if defined(__GCC__)
-#define UNUSED __atribute__((unused))
+#define UNUSED __attribute__((unused))
 #else
 #define UNUSED
 #endif
-- 
1.7.10.4

